CHANGELOG.md

```diff
@@ -5,8 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [v1.2.6] - 2020-01-23
 ### Changed
-- Fix: when using default backups-prepare script, manage the case when no assets_paths are defined
+- Fix: when using default backups-prepare script, manage the case when no assets_paths are defined.
```

Comment:
I would rather list it under a `Fixed` category as described in `Types of changes` at https://keepachangelog.com/en/1.0.0/.

It is rather conmon to search known issues in the changelog to see which version fixed it and this makes it easier.